examples/using-email-channel/app.php

```diff
@@ -46,7 +47,10 @@
         new Email\Channel(
             new Mailer(new GmailSmtpTransport(getenv('MAILER_USERNAME'), getenv('MAILER_PASSWORD'))),
             Email\MessageGenerator::withTimeToDiff(new DateTimeImmutable()),
-            Email\ByPassEmail::sendAllTo(getenv('MAILER_USERNAME'))
+            new Email\AddressGenerator((new ByPassEmail())
+                ->setSendEmailsToAssignee(false) // <- OverriddenEmails wont have effect as long as this is false
```

Comment:
Maybe is a good idea creating a constant at the top of the class or even in the `.env` file with this value instead of this comment in a random place, what do you think?

Reply:
Well, this is a fluent constructor via setters. 
I was thinking about a named constructor instead (I prefer them, actually) but I didn't see the profit of having them in this context, rather than just simply fluent setters... 
I don't like comments as well, but remember, this is a file in the `examples/` directory. It is not part of any production file. It is just an example, that's why that comment there.

Reply:
That's right.
Ok them. :+1: